packages/schematics/src/collection/node-application/files/nestjs/app/app.controller.ts__tmpl__

```diff
@@ -2,7 +2,7 @@ import { Controller, Get } from '@nestjs/common';
 
 import { AppService } from './app.service';
 
-@Controller()
+@Controller('api')
```

Comment:
Maybe we want this on the module so that people can add controllers to the appmodule and they'll all be scoped under `/api`

Reply:
How do I do it on the module?

Reply:
I generally use `app.setGlobalPrefix(`api}`);` in the main.ts